### DISCUSSION TITLE
Do I have to use a framework?

### DISCUSSION THREAD

**Discussion by Bilge** [storybookjs/storybook#14948] [General]
The [tutorial](https://storybook.js.org/tutorials/intro-to-storybook/react/en/get-started/) seems to imply I must use one of: React, React Native, Vue, Angular, Svelte or Ember, but whether or not this is a requirement is never made clear. Can I use it without a framework or not?

**Comment by shilman** [comment]
Each framework provides features for building components in that specific framework. If you don't need any of that, `@storybook/html` is a good choice.

**Reply by Bilge** [reply]
So, to be clear, you don't *actually* need a framework, even though all the tutorials, bar none, make the firm assumption you will be using one and there doesn't seem to be even a single example in the entire rest of the documentation that doesn't use a framework?

**Reply by shilman** [reply]
That's correct. Our tutorials are community contributed and represent the most common use cases for Storybook. If you would like to contribute a tutorial or documentation on `@storybook/html`, I'm sure it would help a lot of people out, and our documentation lead @jonniebigodes would be happy to help get it merged.

While there isn't much `@storybook/html` documentation other than the package README, there  are some code samples here: https://github.com/storybookjs/storybook/tree/next/examples/html-kitchen-sink

**Reply by Bilge** [reply]
How do you imagine I'm in a position to contribute documentation when I don't have the first clue what is even going on?

**Reply by jonniebigodes** [reply]
@Bilge me, @shilman and some other maintainers are currently working on some documentation to help contributors such as yourself to get started contributing to Storybook. Either is for code or documentation. The pull request is currently under review and hopefully some time next week we'll get it merged and documentation released. For your case, native HTML it shouldn't be such an issue to get it documented.
